api/api.py

```diff
@@ -20,6 +20,35 @@ class LiveUpdateStep:
   """
   pass
 
+class PortForward:
+  """
+  Specifications for running and displaying a Kubernetes port-forward.
+
+  By default, the host for a port-forward is ``localhost``. This can be changed with
+  the ``--host`` flag when invoking Tilt via the CLI.
+
+  Attributes:
+    local_port (int): the local port to forward traffic to.
+    container_port (int, optional): if provided, the container port to foward traffic _from_. If not provided, Tilt will forward traffic from ``local_port``, if exposed, and otherwise, from the first default container port. E.g.: ``PortForward(1111)`` forwards traffic from container port 1111 (if exposed; otherwise first default container port) to ``localhost:1111``.
+    link_text (str, optional): if provided, the text of this URL when displayed in the Web UI. This parameter can be useful for disambiguating between multiple port-forwards on a single resource, e.g. labeling one link "App" and one "Debugger." If not given, the Web UI displays the URL itself (e.g. "localhost:8888").
```

Comment:
wonder if we should call this `name` or use some other existing terminology in the kubernetes ecosystem